src/app/routes/edges/routes/PrivateEdgeList/index.js

```diff
@@ -1314,37 +1341,45 @@ class PrivateEdgeList extends React.Component {
                             </div>
                           </TableCell>
                           <TableCell data={n}>
-                            {this.state.userRole !== "READ_ONLY_OPS" && (
-                              <Tooltip title="Kubectl Settings">
-                                <IconButton
-                                  aria-label="edit"
-                                  className="m-0"
-                                  onClick={(event) => {
-                                    this.handleKubectlSettings(null, n);
+                            <div className="row">
+                            { isClusterReady(n) && ( 
```

Comment:
Should we add `this.state.userRole !== "READ_ONLY_OPS"` check similar to Setting and Delete action?

Reply:
not required, sentry jit will take care of translating paralus read roles to corresponding role or clusterrole rbac and user will be able to perform view only kubectl commands.